traffic_portal/app/src/common/modules/form/server/form.server.tpl.html

```diff
@@ -133,20 +133,21 @@
                     <span ng-show="hasError(serverForm.interfaceName)" class="form-control-feedback"><i class="fa fa-times"></i></span>
                 </div>
             </div>
-            <div class="form-group" ng-class="{'has-error': hasError(serverForm.ipAddress), 'has-feedback': hasError(serverForm.ipAddress)}">
+            <div class="form-group" ng-class="{'has-error': hasError(serverForm.ipAddress) && hasError(serverForm.ip6Address), 'has-feedback': hasError(serverForm.ipAddress) && hasError(serverForm.ip6Address)}">
                 <label class="control-label col-md-2 col-sm-2 col-xs-12">Network IP *</label>
```

Comment:
how about adding a tooltip to explain what this new checkbox is for? Kinda like this:

```
                <label class="has-tooltip control-label col-md-2 col-sm-2 col-xs-12">Network IP *<div class="helptooltip">
                    <div class="helptext">Add a description here about what this IP is for and what 'is service address' means</div>
                </div>
                </label>
```

also do that for IPv6?

Reply:
you can find descriptions of each field here: https://traffic-control-cdn.readthedocs.io/en/latest/api/v2/servers.html#post